spec/services/create_escort_spec.rb

```diff
@@ -14,18 +14,32 @@
   end
 
   context 'when there are existing escorts for the given prison number' do
-    let!(:existent_escort) { create(:escort, :completed, prison_number: prison_number) }
+    context 'and escort is not cancelled' do
+      let!(:existent_escort) { create(:escort, :completed, prison_number: prison_number) }
 
-    it 'creates a clone of the most recent escort' do
-      expect(Escort.where(prison_number: prison_number).count).to eq(1)
-      escort = described_class.call(prison_number: prison_number)
-      expect(Escort.where(prison_number: prison_number).count).to eq(2)
-      expect_detainee_to_be_cloned(existent_escort, escort)
-      expect_risk_assessment_to_be_cloned(existent_escort, escort)
-      expect_healthcare_assessment_to_be_cloned(existent_escort, escort)
-      expect_offences_to_be_cloned(existent_escort, escort)
-      expect(escort.move).to be_nil
-      expect(escort.twig).to eq(existent_escort)
+      it 'creates a clone of the most recent escort' do
+        expect(Escort.where(prison_number: prison_number).count).to eq(1)
+        escort = described_class.call(prison_number: prison_number)
+        expect(Escort.where(prison_number: prison_number).count).to eq(2)
+        expect_detainee_to_be_cloned(existent_escort, escort)
+        expect_risk_assessment_to_be_cloned(existent_escort, escort)
+        expect_healthcare_assessment_to_be_cloned(existent_escort, escort)
+        expect_offences_to_be_cloned(existent_escort, escort)
+        expect(escort.move).to be_nil
+        expect(escort.twig).to eq(existent_escort)
+      end
+    end
+
+    context 'and escort is cancelled' do
+      let!(:existent_escort) { create(:escort, :completed, prison_number: prison_number, cancelled_at: Time.now) }
```

Comment:
Please use a trait for `:cancelled`. I've seen the same code repeated now in a lots of specs. DRY.